upstream-sha256: b6469b5f8c1c65a76df5ef3142211e4af302d657a10668a6fe39eb88e9a3ac6b
actual-sha256: efa0cdb83cb6570f0435a8964424372f6bfd8ed68a9ccf0276508b4e0a737cce
--- upstream.expected
+++ tolk.actual
@@ -8,7 +8,7 @@
   __attribute__((shared, aligned(16)))float buf1[16];
   int lidx0 = __ockl_get_local_id(0); /* 16 */
   float val0 = (*(data1_16+lidx0));
-  *(buf1+lidx0) = val0;
+  *(buf1+lidx0) = (0.0f+val0);
   __builtin_amdgcn_fence(__ATOMIC_RELEASE, "workgroup");__builtin_amdgcn_s_barrier();__builtin_amdgcn_fence(__ATOMIC_ACQUIRE, "workgroup");
   *(buf0+0) = 0.0f;
   for (int Ridx101 = 0; Ridx101 < 16; Ridx101++) {
